contrib/gitian-build.sh

```diff
@@ -314,7 +314,8 @@ then
 	    ./bin/gbuild -j ${proc} -m ${mem} --commit pivx=${COMMIT} --url pivx=${url} ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
 	    ./bin/gsign -p $signProg --signer $SIGNER --release ${VERSION}-aarch64 --destination ../gitian.sigs/ ../pivx/contrib/gitian-descriptors/gitian-aarch64.yml
 	    mv build/out/pivx-*.tar.gz build/out/src/pivx-*.tar.gz ../pivx-binaries/${VERSION}
-	popd
+	    popd
+	fi    
```

Comment:
`fi` should come before `popd`

Reply:
Thank you pointing out my mistake using spaces as indents, I've edited it on github editor without realizing it, and sorry for the inconvenience.

Thank you